packages/twenty-server/src/modules/view/standard-objects/view-filter.workspace-entity.ts

```diff
@@ -87,7 +87,7 @@ export class ViewFilterWorkspaceEntity extends BaseWorkspaceEntity {
 
   @WorkspaceField({
     standardId: VIEW_FILTER_STANDARD_FIELD_IDS.positionInViewFilterGroup,
-    type: FieldMetadataType.POSITION,
+    type: FieldMetadataType.NUMERIC,
```

Comment:
use NUMBER

NUMERIC should not exist and is deprecated (there is a confusion between the Postgres column NUMERIC and the metadata field type NUMBER)

Reply:
double check that this won't break anything if we have this code but before sync metadata